packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html

```diff
@@ -0,0 +1,216 @@
+            </oui-switch>
+            <span
+                ><strong
+                    data-ng-bind="$ctrl.getSwitchBtnStatusText($ctrl.model.brotli.enabled)"
+                ></strong
+            ></span>
+        </div>
+
+        <!--CACHE RULE-->
+        <div class="mb-4">
+            <p class="oui-heading_5 mb-1"><strong>Cache rule</strong></p>
+            <div
+                class="oui-paragraph"
+                data-translate="hosting_cdn_shared_option_always_online_info"
+            ></div>
+            <oui-button
+                class="mb-3"
+                data-variant="secondary"
+                icon-left="oui-icon-plus"
+                data-on-click="$ctrl.openCreatCacheRuleModal()"
+            >
+                <span
+                    data-translate="hosting_cdn_shared_option_cache_rule_btn_add_rule"
+                ></span>
+            </oui-button>
+            <div>
+                <table class="oui-table">
+                    <thead class="oui-table__headers">
+                        <tr>
+                            <th
+                                class="oui-table__header oui-table__header_sortable"
+                                tabindex="0"
+                                data-translate="hosting_cdn_shared_option_cache_rule_table_header_order_by"
+                            >
+                                <span
+                                    class="oui-icon oui-icon-sort-inactive"
+                                    aria-hidden="true"
+                                ></span>
+                            </th>
+                            <th
+                                class="oui-table__header"
+                                data-translate="hosting_cdn_shared_option_cache_rule_table_header_rule_name"
+                            ></th>
+                            <th
+                                class="oui-table__header"
+                                data-translate="hosting_cdn_shared_option_cache_rule_table_header_uri"
+                            ></th>
+                            <th
+                                class="oui-table__header"
+                                data-translate="hosting_cdn_shared_option_cache_rule_table_time_to_live"
+                            ></th>
+                            <th
+                                class="oui-table__header"
+                                data-translate="hosting_cdn_shared_option_cache_rule_table_status"
+                            ></th>
+                            <th
+                                class="oui-table__header"
+                                data-translate=""
+                            ></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr
+                            class="oui-table__row"
+                            data-ng-repeat="rule in $ctrl.model.rules | orderBy:'config.priority'"
```

Comment:
```suggestion
                            data-ng-repeat="rule in $ctrl.model.rules track by rule | orderBy:'config.priority'"
```
Or whatever property that seems relevant to track by

Reply:
fixed

Reply:
Seems not.